index.html

```diff
@@ -176,11 +190,117 @@ <h2>Getting Started</h2>
+                </p>
+
+                <div class="pros-and-cons">
+                    <div class="pros">
+                        <h4>Pros</h4>
+                        <ul>
+                            <li>Currently best compression</li>
+                            <li>Simple to use</li>
+                            <li>Support for 32 bits and 64 bits</li>
+                        </ul>
+                    </div>
+
+                    <div class="cons">
+                        <h4>Cons</h4>
+                        <ul class="cons">
+                            <li>Slow</li>
+                        </ul>
+                    </div>
+                </div>
+
+                <div class="links">
+                    <h4>Links</h4>
+                    <ul class="tool-links">
+                        <li><a href="https://logicoma.io/squishy/">squishy project page</a></li>
+                        <li><a href="https://www.pouet.net/prod.php?which=85098">squishy on Pouët</a></li>
+                        <li><a href="https://demozoo.org/productions/295987/">squishy on Demozoo</a></li>
+                        <li><a href="https://github.com/farbrausch/fr_public/tree/master/kkrunchy">kkrunchy source code</a></li>
+                        <li><a href="https://docs.google.com/spreadsheets/d/1tbQ7fgnG5DAI06eFoXvkE8fZKnrO4SpYQrrBbsmwdWk/edit?usp=sharing">Metrics comparing squishy to kkrunchy</a></li>
+                        <li><a href="https://www.youtube.com/watch?v=O5LfE_qNzes">Revision 2020 seminar: modern 64k intro compression</a></li>
+                        <li><a href="https://yupferris.github.io/blog/2020/08/31/c64-4k-intro-packer-deep-dive.html#squishys-decompressor-compressor">Blog post with some details about squishy</a></li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="tool">
+                <h3>Shader Minifier</h3>
+                <p>
+                    Written by Ctrl-Alt-Test, Shader Minifier is the de facto shader minification tool. It rewrites shader source code in a way that uses a smaller footprint and compresses well.
+                </p>
+
+                <div class="pros-and-cons">
+                    <div class="pros">
+                        <h4>Pros</h4>
+                        <ul>
+                            <li>Many minification options</li>
+                            <li>Source code available</li>
+                            <li>Support for GLSL and HLSL</li>
+                        </ul>
+                    </div>
+
+                    <div class="cons">
+                        <h4>Cons</h4>
+                        <ul>
+                            <li>Limited support for multiple shaders</li>
+                        </ul>
+                    </div>
+                </div>
```

Comment:
The pros/cons don't bring a lot of value, we can just mention that it supports GLSL/HLSL in the first paragraph. Apart from that, we can just delete the section.